CBinding/Gui/CTextEditorExtension.cs

```diff
@@ -283,7 +284,7 @@ bool ShouldCompleteOn (char pressed)
 
 		public override Task<ICompletionDataList> HandleCodeCompletionAsync (CodeCompletionContext completionContext, CompletionTriggerInfo triggerInfo, CancellationToken token = default (CancellationToken))
 		{
-			var project = (CProject)DocumentContext.Project;
+			var project = (CMakeProject)(DocumentContext.Project as SolutionItem);
```

Comment:
This will crash. `DocumentContext.Project` isn't castable to `CMakeProject`.

It should probably be:

```
var project = (CMakeProject)DocumentContext.Owner;
```